public/lsp/sidebar.php

```diff
@@ -76,7 +76,8 @@
 					<input type="hidden" name="subcategory" value="<?php echo GET('subcategory');?>" />
 					<input type="hidden" name="oldaction" value="<?php echo GET('action');?>" />
 					</form>
-					<a href="?action=register"><span class="fas fa-chevron-circle-right"></span>&nbsp;Not registered yet?</a>
+					<a href="?action=register"><span class="fas fa-chevron-circle-right"></span>&nbsp;Not registered yet?</a><br />
+					<a href="?account=forget"><span class="fas fa-chevron-circle-right"></span>&nbsp;Forget password?</a>
```

Comment:
I think this should be `Forgot password?`

As per
![image](https://user-images.githubusercontent.com/6282045/66257120-6872ad00-e795-11e9-8006-131ff22f4e92.png)
![image](https://user-images.githubusercontent.com/6282045/66257139-9e179600-e795-11e9-94fc-bd45c1ff8991.png)